deepchem/models/torch_models/tests/highway_test.py

```diff
@@ -0,0 +1,27 @@
+import torch
+import torch.nn as nn
+import torch.nn.init as init
+
+def test_highway():
```

Comment:
You need to add a test that this layer produces the same outputs as the TF layer on the same inputs. See other tests for porting added in recent ports